src/main/java/mil/dds/anet/threads/mart/services/MartReportImporterService.java

```diff
@@ -0,0 +1,402 @@
+package mil.dds.anet.threads.mart.services;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import microsoft.exchange.webservices.data.property.complex.FileAttachment;
+import mil.dds.anet.beans.Attachment;
+import mil.dds.anet.beans.EmailAddress;
+import mil.dds.anet.beans.GenericRelatedObject;
+import mil.dds.anet.beans.Location;
+import mil.dds.anet.beans.Organization;
+import mil.dds.anet.beans.Person;
+import mil.dds.anet.beans.Position;
+import mil.dds.anet.beans.Report;
+import mil.dds.anet.beans.ReportPerson;
+import mil.dds.anet.beans.Task;
+import mil.dds.anet.beans.WithStatus;
+import mil.dds.anet.beans.mart.MartImportedReport;
+import mil.dds.anet.beans.mart.ReportDto;
+import mil.dds.anet.beans.search.LocationSearchQuery;
+import mil.dds.anet.database.AttachmentDao;
+import mil.dds.anet.database.EmailAddressDao;
+import mil.dds.anet.database.LocationDao;
+import mil.dds.anet.database.MartImportedReportDao;
+import mil.dds.anet.database.OrganizationDao;
+import mil.dds.anet.database.PersonDao;
+import mil.dds.anet.database.PositionDao;
+import mil.dds.anet.database.ReportDao;
+import mil.dds.anet.database.TaskDao;
+import mil.dds.anet.database.mappers.MapperUtils;
+import mil.dds.anet.resources.AttachmentResource;
+import mil.dds.anet.utils.ResourceUtils;
+import mil.dds.anet.utils.Utils;
+import org.apache.tika.Tika;
+import org.apache.tika.io.TikaInputStream;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.BeanUtils;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MartReportImporterService implements IMartReportImporterService {
+
+  private static final String SECURITY_MARKING_JSON_PROPERTY = "securityMarking";
```

Comment:
Please move this down below `public static final String REPORT_JSON_ATTACHMENT`.